### ISSUE TITLE
fakeAsync patch for requestAnimationFrame is not passing the timestamp

### ISSUE THREAD

**Issue by santialbo** [angular/zone.js#1216]
When requestAnimationFrame is patched in fakeAsync the callback is not called with a timestamp like it would be expected. Instead it's called with the callback itself.

```
  it('should patch requestAnimationFrame correctly', fakeAsync(() => {
    requestAnimationFrame((timestamp) => {
      // Fails: Expected 'function' to be 'number'.
      expect(typeof timestamp).toBe('number');
    });
    tick(16);
  }));
```

I created this repo with the failing test
https://github.com/santialbo/raf-zonejs-test/blob/4491d675287b2513e39244a368877bb951b19248/src/app/app.component.spec.ts#L32-L38
It's a standard cli generated project with just the failing test

**Comment by JiaLiPassion** [comment]
I will check it, thanks

**Comment by JiaLiPassion** [comment]
@santialbo, thank you, this is a bug, I have fixed it in #1220

**Comment by santialbo-actimo** [comment]
Thank you so much!! do you have any idea when the next release of zone.js will be?
